dusty/payload.py

```diff
@@ -19,10 +20,11 @@ def run(self):
         self.fn(*self.args, **self.kwargs)
 
     def serialize(self):
-        doc = {'fn': self.fn, 'client_version': self.client_version, 'args': self.args, 'kwargs': sorted(self.kwargs.items())}
+        doc = {'fn': self.fn, 'client_version': self.client_version, 'suppress_warnings': self.suppress_warnings,
+               'args': self.args, 'kwargs': sorted(self.kwargs.items())}
         return cPickle.dumps(doc).encode('string_escape')
 
     @staticmethod
     def deserialize(doc):
         original_doc = cPickle.loads(doc.decode('string_escape'))
-        return original_doc['fn'], original_doc['client_version'], original_doc['args'], dict(original_doc['kwargs'])
+        return original_doc['fn'], original_doc['client_version'], original_doc['suppress_warnings'], original_doc['args'], dict(original_doc['kwargs'])
```

Comment:
This pattern is totally shit, can we refactor it while you're here? At least into a dict or something?